cli/setup.py

```diff
@@ -18,13 +18,14 @@
         'future; python_version == "2.7"',
         'spython',
         'click',
-        'requests',
+        'requests == 2.14.2',
```

Comment:
this is a leftover dependency from v1.x. We can remove this as well as `lark-parser` and `pyaes`

Reply:
Was just checking the dependencies, seems like `docker-py` depends on `requests==2.20.0`. From this [requirements](https://github.com/docker/docker-py/blob/master/requirements.txt)

Reply:
yeah, that is handled by `pip` though. So we only need to declare `docker-py` as a dependency here, and pip will handle docker-py's dependencies